routes/api.php

```diff
@@ -13,6 +13,11 @@
 |
 */
 
-Route::middleware('auth:api')->get('/user', function (Request $request) {
-    return $request->user();
+Route::post('/login', 'Api\Auth\LoginController@login');
+Route::post('/login/refresh', 'Api\Auth\LoginController@refresh');
+Route::post('/register', 'Api\Auth\RegisterController@register');
+
+Route::group(['middleware' => 'auth:api'], function(){
+    Route::post('/logout', 'Api\Auth\LoginController@logout');
+    ROute::get('/user', 'Api\UserController@index');
 });
```

Comment:
Use constant for route URI ?

Reply:
Apparently this is not the way things are done in Laravel world but that would make sense to me.